std_msgs/Header Header

string robot_name
KeyFrameRos[] keyframes
EdgeRos[] edges